TODO Api/app/routes/router.py

```
@@ -0,0 +1,94 @@
+
+
+# ------------------------------ Post Requests------------------------------
+
+
+@router.post("/", status_code=201)
+async def create_task(task: Task, db: Session = Depends(get_db)):
+    try:
+        task_model = models.Tasks(**task.dict())
+
+        db.add(task_model)
+        db.commit()
+        return JSONResponse(content="Successfully created the task")
+    except Exception as e:
+        print(e)
+        raise HTTPException(status_code=400, detail=f"Some error occured.")
+
+
+# ------------------------------ Put Requests------------------------------
+
+
+@router.put("/{task_id}", status_code=200)
+async def update_task(task_id: int, task: TaskDetails, db: Session = Depends(get_db)):
+    try:
+        task_model = db.query(models.Tasks).filter(models.Tasks.id == task_id).first()
+        if task_model is None:
+            raise HTTPException(status_code=404, detail="ID doesn't exist")
+        task_model.name = task.name
+        task_model.description = task.description
+        task_model.priority = task.priority
+
+        db.add(task_model)
+        db.commit()
+        return JSONResponse(content="Successfully updated the task")
+    except:
+        return HTTPException(status_code=404, detail=f"Task Not found.")
+
+
+# ------------------------------ Delete Requests------------------------------
+
+
+@router.delete("/{task_id}", status_code=200)
+async def delete_task(task_id: int, db: Session = Depends(get_db)):
+    try:
+        task_model = db.query(models.Tasks).filter(models.Tasks.id == task_id).first()
+        if task_model is None:
+            raise HTTPException(status_code=404, detail="ID doesn't exist")
+        db.query(models.Tasks).filter(models.Tasks.id == task_id).delete()
+        db.commit()
+        return JSONResponse(content="Successfully deleted the task")
+    except:
+        return HTTPException(status_code=404, detail=f"Task Not found.")
```

This is not going to work. You are returning an `Exception`. Exceptions are never returned only raised. This not going to work as expected and result in some really bad bugs.